### ISSUE TITLE
test second iteration of function in a loop

### ISSUE THREAD

**Issue by froz42** [froz42/funcheck#88]
**Is your feature request related to a problem? Please describe.**
If an allocation in used more that one time in a loop for example the program will not detect the probleme in the second iteration

**Describe the solution you'd like**
will test second iteration if the usage count is > 1 

**Additional context**
It should detect the memory leak in this code:

```c
char *malloc_function(void)
{
	return (malloc(1000));
}

int main(void)
{
	void *ptr = malloc_function();
	if (ptr == NULL)
		return (1);
	free(ptr);

	void *array[10];

	for (int i = 0; i < 10; i++)
	{
		array[i] = malloc(1000);
		if (array[i] == NULL)
		{
			printf("it fail %i\n", i);
			return (0);
		}
		memset(array[i], 0, 1000);
	}
	
	for (int i = 0; i < 10; i++)
		free(array[i]);

	ptr = malloc_function();
	memset(ptr, 0, 1000);
	free(ptr);
}
```
